### PR TITLE
dep(types): typescript and typings dependencies

### PR DESCRIPTION
**Pull request by cnishina**
- remove typings dependency, scripts, and gulp task
- add @types dependencies
- clean up globals.ts from npm publishing

### CODE REVIEW

**Comment by cnishina** [comment]
@robinboehm I plan to rollback your change. The transpile to `"outDir": "tmp"` when upgrading to TypeScript 2 would output javascript files to the path: `tmp/protractor/exampleTypescript`. I am open to suggestions.

I did however update the Protractor Cookbook with the `outDir` change: https://github.com/angular/protractor-cookbook/commit/e2b823eb33720ab1aad290657140c5f4de2b25c2

**Comment by cnishina** [comment]
@robinboehm just kidding... I was able to get this to work with @tmeneau's help.

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,4 @@ npm-debug.log
 
 *.swp
 globals.js
+globals.d.ts
```
Comment (sjelin) [inline .gitignore]: Thank g-d

**Comment by mgiambalvo** [comment]
yay no more typings.json!

**Pull request merged**